File: python/pad_array.py
#REMEMBER TO PSEUDOCODE
def pad(array, min_size, value = None):
    if min_size > len(array):
        for i in range(min_size-len(array)):
            array.append(value)
    else:
        return array
    
    return array


# print(pad([1,2,3], 5))
# print(pad([1,2,3], 5, 'apple'))
# print(pad([1,2,3], 1))

File: js/padArray.js
// REMEMBER TO PSEUDOCODE
const pad = (array, minSize, value=null) => {
    if (minSize <= array.length) return array
    padCount = minSize - array.length;
    for (let i = 1; i <= padCount; i++) {
        array.push(value)
    }
    return array
}

module.exports = { pad }

